wuzz.go

```diff
@@ -1394,3 +1426,10 @@ func main() {
 		log.Panicln(err)
 	}
 }
+
+var tlsVersions = map[string]uint16{
+	"SSL3.0": tls.VersionSSL30,
+	"TLS1.0": tls.VersionTLS10,
+	"TLS1.1": tls.VersionTLS11,
+	"TLS1.2": tls.VersionTLS12,
+}
```

Comment:
Global variables are upper case and defined in the top of the file, please follow these conventions.